### ISSUE TITLE
Ped spawn dont sync

### ISSUE THREAD

**Issue by RoykeHD** [oosayeroo/qb-miniheists#7]
Players have the issue if they do the Merry weather heist or the lab heist that other players cant see the peds.
Some peds are sync but some are not.

**Comment by AllRoundJonU** [comment]
Hey, as you can see, I am not the owner of this script, but I made a version of qb-miniheists that contains some improvements.
My version currently not contain all the heists from this script, but will come.

If you are still looking for a fix for your problem, fell free to use my version of the script [it-smallheists](https://github.com/inseltreff-net/it-smallheists/tree/master)

**Comment by Rosskho** [comment]
Yes mine dont sync either! I can see enemies but my teamates dont.
